hpotter/env.py

```diff
@@ -8,9 +8,9 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker, scoped_session
 from sqlalchemy_utils import database_exists, create_database
-from hpotter.tables import Base
+from sqlalchemy.ext.declarative import declarative_base, DeferredReflection
 
-logging.config.fileConfig('hpotter/logging.conf')
+logging.config.fileConfig('/Users/jeffrowell/Desktop/HPotter/hpotter/logging.conf')
```

Comment:
Pointing to your file structure Jeff, not sure if we need to change this right away just did not want it to go unnoticed.